libs/libcommon/src/libcommon/config.py

```diff
@@ -42,12 +42,14 @@
 
 ASSETS_BASE_URL = "assets"
 ASSETS_STORAGE_DIRECTORY = None
+ASSETS_S3_FOLDER_NAME = "assets"
```

Comment:
it's a bit weird to have two configuration variables (ASSETS_STORAGE_DIRECTORY and ASSETS_S3_FOLDER_NAME), while we will only use one of them.

But I understand that this will be fixed in another PR where we will abstract S3 or FS storages in a common interface.

Reply:
Yes, I think using `fsspec` would help us create the file directly in a specified location and ASSETS_STORAGE_DIRECTORY will be removed.